jsf/subsystem/src/main/java/org/jboss/as/jsf/deployment/JSFComponentProcessor.java

```diff
@@ -173,6 +176,69 @@ private void processXmlManagedBeans(final DeploymentUnit deploymentUnit, final S
         }
     }
 
+    /**
+     * WFLY-6617 workaround
```

Comment:
Tiny issue but this is not a workaround, its just how we implement component injection (not enough of an issue to hold up merging).